ui/src/actions/user.js

```diff
@@ -137,6 +138,13 @@ export function setPreferredCiteFormat(format) {
   };
 }
 
+export function setIsCitationSummaryEnabled(isCitationSummaryEnabled) {
```

Comment:
I was just thinking if it would be a good idea to have `preferences: { }` in `user` state, instead of creating actions for each of them.

Reply:
so you want to move also ```preferredCiteFormat``` to these preferences?